### PR TITLE
#969 write test for webhook module

### PR DESCRIPTION
**Pull request by minhtranq-nashtechglobal**

### CODE REVIEW

File: webhook/pom.xml [hunk 1]
```diff
@@ -47,6 +47,16 @@
 			<groupId>org.liquibase</groupId>
 			<artifactId>liquibase-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
```
Comment (khanhtrand) [inline webhook/pom.xml:51]: I think this dependency is unused.

Comment (minhtranq-nashtechglobal) [reply webhook/pom.xml:51]: I use this dependency for integration test

Comment (khanhtrand) [reply webhook/pom.xml:51]: For integration test, I think you are using PostgreSQL DB not H2 DB. Is it correct?

`@Bean(destroyMethod = "stop")
    public PostgreSQLContainer<?> postgresContainer() {
        return new PostgreSQLContainer<>("postgres:16")
            .withReuse(true);
    }`

Comment (minhtranq-nashtechglobal) [reply webhook/pom.xml]: Fixed

**Review by khanhtrand — APPROVED** [review]

**Review by VanCongBang — APPROVED** [review]

**Pull request merged**